app/helpers/sit_validate_participant_helper.rb

```diff
@@ -40,4 +40,8 @@ def previous_induction?(profile)
   def participant_has_no_qts?(profile)
     profile.ecf_participant_eligibility&.no_qts_reason?
   end
+
+  def display_name(form)
+    form.type == :self ? "your" : "#{form.full_name&.titleize}’s"
+  end
```

Comment:
Could this be a method on the form instead?
i.e. `form.display_name` seems a bit cleaner

Reply:
Thanks :) no problem, i'll move it over